lib/keila_web/helpers/normalizer/normalizer.ex

```diff
@@ -0,0 +1,93 @@
+defmodule KeilaWeb.ApiNormalizer do
+  import Plug.Conn
+  import Phoenix.Controller, only: [render: 2]
+  import KeilaWeb.ApiNormalizer.SchemaMapper
+
+  def init(opts) do
+    opts
+  end
+
+  def call(conn, opts) do
+    opts
+    |> Keyword.get(:normalize, [])
+    |> Enum.reverse()
```

Comment:
Is this really necessary? What is the purpose of reversing the normalize options?